components/Header/Header.tsx

```diff
@@ -19,6 +19,11 @@ const NAVIGATION_LINKS = [
         title: 'Digital marketing',
         url: 'https://torchbox.com/digital-marketing/',
     },
+    {
+        title: 'AI + Innovation',
+        url: 'https://ai.torchbox.com/',
+        isExternalLink: true,
+    },
```

Comment:
Chore: Please can you add the new `work` link included on the main torchbox site to this array.

Work - https://torchbox.com/work/

Reply:
done